symphony/content/content.systemauthors.php

```diff
@@ -695,7 +707,15 @@ public function __actionEdit()
 
             if ($this->_Author->isPrimaryAccount() || ($isOwner && Symphony::Author()->isDeveloper())) {
                 $this->_Author->set('user_type', 'developer'); // Primary accounts are always developer, Developers can't lower their level
-            } elseif ((Symphony::Author()->isDeveloper() || Symphony::Author()->isManager()) && isset($fields['user_type'])) {
+            } elseif (Symphony::Author()->isManager() && isset($fields['user_type'])) { // Manager can only change user type for author and managers
+
+                if ($fields['user_type'] !== 'author' && $fields['user_type'] !== 'manager') {
+                    $this->pageAlert(__('The user type is invalid. You can only set authors and manager.'), Alert::ERROR);
+                    return;
+                }
+
+                $this->_Author->set('user_type', $fields['user_type']);
+            } elseif (Symphony::Author()->isDeveloper() && isset($fields['user_type'])) {
```

Comment:
I would also add this check if an author wants to edit someone and throw if is does.